plugin/base.go

```diff
@@ -18,15 +18,17 @@ var LogGroupName string
 var LogStreamName string
 var FunctionARN string
 
-var TraceId string
-var TransactionId string
-var SpanId string
+var TraceID string
+var TransactionID string
+var SpanID string
 
 var ApiKey string
 
+var CtxWithRootSpan context.Context
```

Comment:
There should not be trace specific global variable here as this is common place. We need to find better approach.